src/pages/MainPage/MainPage.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+//Components
+import Button from "сomponents/Button";
+//helper
+import { mockedFunction } from "helpers";
+//style
+import style from "./style.module.scss";
+
+const MainPage = () => {
+  return (
+    <div>
+      <div className={style.box}>
+        <div className={style.tittle}>MinesWeeper</div>
+        <Button
+          styles={style.btnStart}
+          value="Розпочати гру"
+          to="/createOwnLevel"
+          onClick={() => mockedFunction}
```

Comment:
add this mocked function to Button Component as default onClick event.
And in other places where u put i u can remove that option